packages/core/src/model.js

```diff
@@ -1916,6 +1917,7 @@ ${associationOwner._getAssociationDebugList()}`);
    */
   static async create(values, options) {
     options = cloneDeep(options) ?? {};
+    options.include ??= getDefaultCreateInclude(this, values);
```

Comment:
I think belongs inside of `save` when `isNewRecord` is true, because calling build then save should work too